frontend/.env.example

```diff
@@ -50,7 +50,7 @@ AUTH_RASCL_LOGOUT_URL=http://localhost:3000/
 
 # Application feature flags -- see ./app/utils/env.server.ts for valid values
 # (optional; default: "")
-ENABLED_FEATURES=doc-upload,email-alerts,hcaptcha,view-personal-info,view-applications,view-letters,view-messages,edit-personal-info,status
+ENABLED_FEATURES=doc-upload,email-alerts,hcaptcha,view-personal-info,view-applications,view-letters,view-messages,edit-personal-info,status,status-check
```

Comment:
Might be a nitpick but we should consider renaming the `status-check` feature flag to make it obvious this one is for the protected/authenticated space (same for `status`).